### PR TITLE
Create bmetric.eno pattern

### PR DESCRIPTION
**Pull request by david-bmetric**
Created bmetric pattern and added details

### CODE REVIEW

**Comment by david-bmetric** [comment]
I've just added the pattern file with the hope that it can be added to Ghosterys database and that bmetric's script won't be displayed as unknown.

**Comment by philipp-classen** [comment]
Thank you for the pull request. Do you have some examples websites where these requests are being sent? To get a better understanding of the impact of blocking them.

Also, I wonder if the pattern would not rather be Site Analytics (see https://github.com/ghostery/trackerdb/blob/main/docs/categories.md)? But I have no more context, so I'm guessing.

**Comment by david-bmetric** [comment]
Hi @philipp-classen, thanks for your feedback, knowing our product well i think that Customer management fits quite well :). 

Some of our where you can see the script in action are: 
https://tryg.com/en
https://yousee.dk
https://www.telia.dk/

Our own website: https://bmetric.com/ where you can also learn more

Let me know if you have any ore questions :)

File: db/patterns/bmetric.eno [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: bmetric
+category: customer_interaction
+website_url: https://bmetric.com/
+organization: bmetric
+
+--- domains
+
```
Comment (philipp-classen) [inline db/patterns/bmetric.eno:7]:
```suggestion

```

File: db/patterns/bmetric.eno [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+web.telemetric.dk
+insight.bellmetric.net
+bmetricpublisher.com
+
```
Comment (philipp-classen) [inline db/patterns/bmetric.eno:11]:
```suggestion

```

File: db/patterns/bmetric.eno [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+--- domains
+
+--- filters
+
```
Comment (philipp-classen) [inline db/patterns/bmetric.eno:15]:
```suggestion

```

File: db/patterns/bmetric.eno [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+||web.telemetric.dk/t/c/.*
+||insight.bellmetric.net/static/js/publisher.min.js
+||insight.bellmetric.net/vipbar/getnumber/.*
+||bmetricpublisher.com/t/.*/bm.js
+||bmetricpublisher.com/t/.*/tm.js
+||bmetricpublisher.com/t/.*/n?.*
+||bmetricpublisher.com/t/c/.*
+||bmetricpublisher.com/static/js/publisher.min.js
+||bmetricpublisher.com/vipbar/getnumber/.*
+
```
Comment (philipp-classen) [inline db/patterns/bmetric.eno:28]:
```suggestion

```

**Review by philipp-classen — CHANGES_REQUESTED** [review]

**Review by philipp-classen — APPROVED** [review]

**Comment by philipp-classen** [comment]
Thank you!

FYI: I removed some blank lines, which broke the tests, and fixed a typo in the organization file.

**Pull request merged**